src/py/flwr/client/app.py

```diff
@@ -365,7 +367,23 @@ def _load_app() -> Flower:
                 app: Flower = load_flower_callable_fn()
 
                 # Handle task message
-                out_message = app(message=message, context=context)
+                try:
+                    out_message = app(message=message, context=context)
+                except Exception as ex:  # pylint: disable=broad-exception-caught
+                    log(ERROR, "FlowerCallable raised an exception", exc_info=ex)
```

Comment:
Should we do the try-except here or in FlowerCallable?

Reply:
Good question - both are viable options, but I'm leaning towards keeping it here in case some user code (perhaps accidentally) modifies the `FlowerCallable` and causes it to raise still an `Exception` that should have been caught.